fenix/app/src/main/java/org/mozilla/fenix/search/toolbar/ToolbarView.kt

```diff
@@ -172,7 +172,11 @@ class ToolbarView(
                 if (!searchEngine.isGeneral) {
                     context.getString(R.string.application_search_hint)
                 } else {
-                    context.getString(R.string.search_hint)
+                    if (searchEngine == searchState.defaultEngine) {
```

Comment:
we want to load urls only by the default search engine, other engines will perform a search with the given url